doc/nrf/config_and_build/multi_image.rst

```diff
@@ -83,7 +83,7 @@ When building the parent image, you can configure how the child image should be
 When building the child image from the source or using a prebuilt HEX file, the build system merges the HEX files of both the parent and child image, so that they can be programmed in one single step.
 This means that you can enable and integrate an additional image just by using the default configuration.
 
-To change the default configuration and configure how a child image is handled, locate the BUILD_STRATEGY configuration options for the child image in the parent image configuration.
+To change the default configuration and configure how a child image is handled, locate the ``BUILD_STRATEGY`` Kconfig options for the child image in the parent image configuration.
```

Comment:
I would consider moving this part to next to `_ug_multi_image_variables:`. I think it would be great to have common part of documentation describing child images configuration from application perspective (so that application developer would know how to configure the bootloader and used net core image - building them as child images is done by the NCS, there is no need to add them manually).

Reply:
Are we going to open a follow-up ticket for this one?